<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Nilesh-Bhavsar-25" failures="1" tests="2" name="Default test(failed)(failed)" time="0.176" errors="0" timestamp="2022-04-19T22:31:10 IST">
  <testcase classname="demo.TestNGRetryFailDemo" name="test2" time="0.038">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailDemo.test2(TestNGRetryFailDemo.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="demo.TestNGRetryFailDemo" name="test3" time="0.003"/>
  <testcase classname="demo.TestNGRetryFailDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed)(failed) -->
